v63003/u_inref/gtm8842.csh

```diff
@@ -13,20 +13,33 @@
 #
 # Tests TRIGGER_MOD restricts ZBREAKS and ZSTEPS
 #
+
+$gtm_tst/com/dbcreate.csh mumps 1 >>& dbcreate.out
+echo "# Uploading triggerfile"
 cat > triggers.txt << EOF
 +^X -command=set -name=triggered -xecute="do trigger^gtm8842"
 EOF
-# Since we cannot write to ydb_dist, we are creating a pseudo ydb_dist called temp that contains
+$MUPIP trigger -triggerfile=triggers.txt
+# Since we cannot write to ydb_dist, we are creating a pseudo ydb_dist under ydb_temp_dist that will contain
 # all the relevant files in ydb_dist and restrict.txt
-mkdir temp
-cp $ydb_dist/* temp/ >>& copy.out
-setenv ydb_dist temp
+mkdir ydb_temp_dist
+cp $ydb_dist/* ydb_temp_dist/ >>& copy.out
```

Comment:
Can you move lines 25 to 26 into a test framework script (say com/copy_ydb_dist_dir.csh) that does this. This way other tests which need a copy can use this rather than duplicating this code.